uk.co.compucorp.civicrm.hrleaveandabsences/xml/option_groups/leave_and_absences_work_pattern_change_reason_install.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
```

Comment:
For the name of this file, please follow the same pattern used for the other files in this folder, where the file name if the option group name without the hrleaveandabsences_ prefix.

Reply:
This will be updated.